src/buildercore/bootstrap.py

```diff
@@ -150,13 +150,17 @@ def unsub_sqs(stackname, new_context, region, dry_run=False):
             policy = json.loads(queue.attributes.get('Policy', '{}'))
             LOG.info("Saving new Policy for %s removing %s (%s)", queue_name, topic_arns, policy)
             new_policy = remove_topics_from_sqs_policy(policy, topic_arns)
-            new_policy = new_policy or ''
```

Comment:
we need to cover this, probably an integration test involving a stack with SQS is needed. The service itself (SQS) is widely used.

Reply:
agreed. I recall it being buggy in the recent boto bump as well